eth_utils/curried/__init__.pyi

```diff
@@ -1,86 +1,5 @@
 from typing import Any, Callable, Sequence, Tuple, TypeVar, Union, overload
 
-from eth_utils.curried import (
-    ExtendedDebugLogger,
-    HasExtendedDebugLogger,
-    HasExtendedDebugLoggerMeta,
-    HasLogger,
-    HasLoggerMeta,
-    ValidationError,
-    add_0x_prefix,
-    apply_formatter_at_index,
-    apply_formatter_if,
-    apply_formatter_to_array,
-    apply_formatters_to_dict,
-    apply_formatters_to_sequence,
-    apply_key_map,
-    apply_one_of_formatters,
-    apply_to_return_value,
-    big_endian_to_int,
-    clamp,
-    combine_argument_formatters,
-    combomethod,
-    decode_hex,
-    denoms,
-    encode_hex,
-    event_abi_to_log_topic,
-    event_signature_to_log_topic,
-    flatten_return,
-    from_wei,
-    function_abi_to_4byte_selector,
-    function_signature_to_4byte_selector,
-    get_extended_debug_logger,
-    get_logger,
-    hexstr_if_str,
-    humanize_hash,
-    humanize_ipfs_uri,
-    humanize_seconds,
-    import_string,
-    int_to_big_endian,
-    is_0x_prefixed,
-    is_address,
-    is_binary_address,
-    is_boolean,
-    is_bytes,
-    is_canonical_address,
-    is_checksum_address,
-    is_checksum_formatted_address,
-    is_dict,
-    is_hex,
-    is_hex_address,
-    is_hexstr,
-    is_integer,
-    is_list,
-    is_list_like,
-    is_normalized_address,
-    is_null,
-    is_number,
-    is_same_address,
-    is_string,
-    is_text,
-    is_tuple,
-    keccak,
-    remove_0x_prefix,
-    replace_exceptions,
-    reversed_return,
-    setup_DEBUG2_logging,
-    sort_return,
-    text_if_str,
-    to_bytes,
-    to_canonical_address,
-    to_checksum_address,
-    to_dict,
-    to_hex,
-    to_int,
-    to_list,
-    to_normalized_address,
-    to_ordered_dict,
-    to_set,
-    to_text,
-    to_tuple,
-    to_wei,
-)
-
```

Comment:
Why was this removed?  Won't this break all of the `from eth_utils.curried` imports?

Reply:
(notice this is the `pyi` file, not the `py` file)

mypy was failing with warnings, on that import. I don't really know the pyi format, but no tests broke when I removed it. :man_shrugging: 

Here's the test with all the warnings:
https://circleci.com/gh/ethereum/eth-utils/2623?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

From my quick read of https://www.python.org/dev/peps/pep-0484/#stub-files I think there's no reason/need for those imports to be there.